configs/ttalps_analysis/ttalps_plotter_config.py

```diff
@@ -75,8 +79,14 @@
 }
 
 default_norm = NormalizationType.to_lumi
+# default_norm = NormalizationType.to_background
 # default_norm = NormalizationType.to_data
 
+plots_from_LLPNanoAOD = False
+
+if(plots_from_LLPNanoAOD):
```

Comment:
No parentheses needed in python ;) (and my VSCode will complain about redundant parentheses - yours should too!)